gatsby-config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   siteMetadata: {
     logo: 'https://www.edx.org/sites/default/files/open-edx-logo-with-reg.png',
     siteName: 'Open Edx',
+    programUUID: '057ca1fb-558c-44b6-84b5-c367eaedf662',
```

Comment:
i am going to see if i can get an answer on where this should come from - i don't know if it would be part of the gatsby config or not.

Reply:
i have gotten confirmation that it will be in the gatsby config.